zappa/core.py

```diff
@@ -315,13 +316,18 @@ def __init__(
         else:
             self.manylinux_suffix_start = "cp39"
 
+        self.architecture = architecture
```

Comment:
Should we validate the given value here?
(It's probably better to identify and throw an error before we try to update aws)

Reply:
Agree. What kind of error should we raise? I would say an `EnvironmentError` would be ok for this purpose.

Reply:
Or `ValueError`.